eZ/Publish/Core/Persistence/Cache/Adapter/TransactionItem.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Persistence\Cache\Adapter;
```

Comment:
I'd expect in this namespace adapters only, so I'm not convinced it's a good place for `TransactionItem`. Maybe rather:
```suggestion
namespace eZ\Publish\Core\Persistence\Cache\Transaction;
```
?
But I haven't explored deeper this architecture, so not insisting, just sayin' "maybe" ;)

Reply:
What about in 
`namespace eZ\Publish\Core\Persistence\Cache\Adapter\Item;`

Then it's still clear it's only useful within the given adapter, and hopefully covers your concern?

Reply:
> What about in
> `namespace eZ\Publish\Core\Persistence\Cache\Adapter\Item;`
> Then it's still clear it's only useful within the given adapter, and hopefully covers your concern?

:+1:  could be like that :)